pkg/http/utils_test.go

```diff
@@ -82,3 +100,20 @@ func TestHttp_Interpolation(t *testing.T) {
 	assert.True(t, matchPath)
 	assert.True(t, matchBody)
 }
+
+func createTempTile(content string) string {
```

Comment:
a bit of duplication. maybe we can put this somewhere common...

Reply:
Is there a typo btw? `createTempTile` -> `createTempFile`?

Reply:
yeap. fixed. i also moved this function to a common package. it's under an "internal" dir which for golang means it will not be accessible outside of mittens.